src/v.ts

```diff
@@ -0,0 +1,148 @@
+const FILE_OR_FOLDER: Fig.Arg = {
+  name: "file",
+  description: "The input file",
+  template: "filepaths",
+};
+
+const completionSpec: Fig.Spec = {
+  name: "v",
+  description: "The V Programming Language",
```

Comment:
See https://fig.io/docs/reference/subcommand#parserdirectives
```suggestion
  description: "The V Programming Language",
  parserDirectives: {
    flagsArePosixNoncompliant: true,
  },
```